tsconfig.json

```diff
@@ -1,111 +1,36 @@
 {
```

Comment:
This just removes the unused params from the tsconfg file....no actual changes, except for removing comments.